app/components/contest/index.js

```diff
@@ -0,0 +1,170 @@
+import Section from '../common/section';
+import { styles, mediaQueries } from '../../constants';
+
+const ContestMain = () => (
+  <Section section={{ style: { backgroundColor: styles.mainColor3 } }}>
```

Comment:
`{ style: { backgroundColor: styles.mainColor3 } }` can be removed. I checked and it doesn't look like it's needed, page background is already white.

Reply:
fixed